### PR TITLE
[breaking change] Trunc div and true div

### PR DESCRIPTION
**Pull request by zasdfgbnm**
Fixes https://github.com/NVIDIA/Fuser/issues/328
- `ops.div` semantics changed from true division to trunc division
- `ops.truediv` added for true division
- version bumped

### CODE REVIEW

**Comment by zasdfgbnm** [comment]
!build

**Comment by zasdfgbnm** [comment]
!build

**Comment by zasdfgbnm** [comment]
!build

**Comment by zasdfgbnm** [comment]
!build

File: test/test_gpu_fused_reduction.cpp [hunk 1]
```diff
@@ -1279,10 +1279,10 @@ TEST_F(NVFuserTest, FusionGroupAllreduce5_CUDA) {
   fe.compileFusion(&fusion, aten_inputs);
   auto outputs = fe.runFusion(aten_inputs);
 
-  auto t3 = t0 / t0.sum({0}).unsqueeze(0).to(at::kComplexDouble);
```
Comment (naoyam) [inline test/test_gpu_fused_reduction.cpp:1282]: Is this a problem? I think I did this type cast to reduce rounding errors.

File: test/test_gpu3.cpp [hunk 2]
```diff
@@ -8567,6 +8567,43 @@ TEST_F(NVFuserTest, AlignedSyncReduction1_CUDA) {
       kernel_string);
 }
 
+// Simple test to check if the aligned block sync is used in aligned
```
Comment (naoyam) [inline test/test_gpu3.cpp:8570]: This is clearly not about aligned block sync 😄

Comment (zasdfgbnm) [reply test/test_gpu3.cpp:8570]: ..... Yeah, so copy pasted this but forget to change🤦

File: test/test_gpu_fused_reduction.cpp (hunk 1 above)
Comment (zasdfgbnm) [reply test/test_gpu_fused_reduction.cpp:1282]: I don't see any failure, but I just added it back just in case.

Comment (naoyam) [reply test/test_gpu_fused_reduction.cpp:1282]: Rounding errors could depend on devices. I tend to do this anytime reduction is involved.

**Review by naoyam — APPROVED** [review]
LGTM

**Pull request merged**